WordPress/Classes/Utility/Migrator/CoreDataIterativeMigrator.swift

```diff
@@ -4,11 +4,60 @@ import CoreData
 /// CoreDataIterativeMigrator: Migrates through a series of models to allow for users to skip app versions without risk.
 ///
 class CoreDataIterativeMigrator: NSObject {
+    static let versionInfoPlist = "VersionInfo.plist"
+    static let versionHashesKey = "NSManagedObjectModel_VersionHashes"
 
     private static func error(with code: IterativeMigratorErrorCodes, description: String) -> NSError {
         return NSError(domain: "IterativeMigrator", code: code.rawValue, userInfo: [NSLocalizedDescriptionKey: description])
     }
 
+    @objc
+    static func migrateStoreToLatestModel(storeURL: URL, modelURL: URL) {
```

Comment:
Is this method for future use? Looks like it's not called anywhere.

Reply:
Nopes.  This was actually part of an approach I was taking to resolve an issue in my PR, but I resolved it another way, so I should remove this code.  Good catch.